go/oasis-node/cmd/genesis/genesis.go

```diff
@@ -618,7 +618,14 @@ func doDumpGenesis(cmd *cobra.Command, args []string) {
 
 	client := consensus.NewConsensusClient(conn)
 
-	doc, err := client.StateToGenesis(ctx, viper.GetInt64(cfgBlockHeight))
+	height, err := cmd.Flags().GetInt64(cfgBlockHeight)
```

Comment:
Oh it seems this is due to addition of `height` to the `cmd/stake/account` command which then conflicts with this `height`.